compiler-rt/test/hwasan/TestCases/memcmp.cpp

```diff
@@ -25,6 +25,5 @@ int main(int argc, char **argv) {
   // CHECK: HWAddressSanitizer: tag-mismatch on address
   // CHECK: MemcmpInterceptorCommon
```

Comment:
interceptor is internal detail, we should not check against it
also android does not have interceptors

however user relevant symbol is main, and there is line number, we should check those

Reply:
and you can break optimizations with volatile vars